### ISSUE TITLE
Is award amount reported a flow or stock data?

### ISSUE THREAD

**Issue by ecyiing** [fedspendingtransparency/USASpending-Issue-Tracker#80]
Hi, I'm new to this dataset and I am wondering if the amount of federal spending reported here refers to the amount of funds actually dispensed in that fiscal year or the total value of the (potentially multi-year) award that is awarded for that year. 

For example, the "dollarsobligated" variable for contracts is defined as "The net dollar amount that is obligated or deobligated by this transaction". This seems to imply that the summation of this variable across transactions for a year would not result in double counting and would reflect the actual flow of funds from the federal government for that year.

However, "Funds Awarded for a Contract" is defined as "A definite commitment by the federal government for the payment of goods and services ordered or received.  An agency makes an award (obligation) when it places an order, signs a contract, awards a grant, purchases a service or takes other action that requires the government to make payments to the public or from one government to another.". In this definition, a government incurs an obligation when it so much as places an order. As such, if this is the amount that is reported under "dollarsobligated", this could result in potential double counting.

In a nutshell, I'm wondering if I should treat the data reported here as a flow or stock data. The difference in the interpretation of this data could potentially be significant if there are large amounts of multi-year contracts/grants awarded by the Federal Government. 

Thank you.

**Comment by USAspendingkevin** [comment]
First, please understand that USAspending.gov downloads contract data from FPDS-NG.  As such it is not the authoritative source for contracts, FPDS-NG is the authoritative source. Thus the operators of FPDS-NG should be the ones to provide definitive field definitions.  That being said, I'll try to provide my insight.  The obligated amount does not represent a flow of funds and does not represent payments made.  It does represent the total of the funds that could be paid. In those instances where several transactions are present, those transactions represent additional obligations or de-obligations.

**Comment by ecyiing** [comment]
Thank you for the clarification @USAspendingkevin. That has been most helpful and I'll contact FPDS-NG for further clarification, if required. Thanks again.
